client/app/views/steps/password.coffee

```diff
@@ -32,6 +36,29 @@ module.exports = class PasswordView extends StepView
         @$visibilityIcon.attr 'xlink:href', data.visibilityIcon
 
 
+    initialize: (args...) ->
+        super args...
+        # lowest level is 1 to display a red little part
+        @passwordStrength = {percentage: 1, label: 'weak'}
```

Comment:
Shouldn't the strenght be given by a call to passwordHelper, to be sure to have the same logic as the one implemented in the helper ?